packages/algorand-js/src/runtime/account.ts

```diff
@@ -75,22 +66,14 @@ export class StoreAccount implements StoreAccountI {
    * @param key: key to fetch value of from local state
    * @param value: key to fetch value of from local state
    */
-  updateLocalState (appId: number, key: Uint8Array, value: StackElem): AppLocalState {
+  updateLocalState (appId: number, key: Uint8Array, value: StackElem): AppLocalStateM {
     const localState = this.appsLocalState.get(appId);
-    const data = getKeyValPair(key, value); // key value pair to put
-
-    const localApp = localState?.["key-value"];
+    const localApp = localState?.[keyValue];
     if (localState && localApp) {
-      const idx = localApp.findIndex(schema => compareArray(schema.key, key));
+      localApp.set(key.toString(), value);
+      localState[keyValue] = localApp; // save updated state
```

Comment:
can we rename keyValue to `localState` or `localStateMap` or something else ?

Reply:
yes. updating to `localStateMap`.